src/components/ebay-checkbox/template.marko

```diff
@@ -10,6 +10,7 @@
         w-onchange="handleChange"
         w-onfocus="handleFocus"
         disabled=data.disabled
+        id=(data.id || widget.id + "-input")
```

Comment:
The idiomatic way to do this in Marko 3 would be `data.id || widget.elId("input")`. Otherwise this LGTM.

Reply:
I coped this from the combobox PR. Either way, not going to port this to marko 4 so I'll leave it like this for now
But for future ones, I'll do it like this 👍